### DISCUSSION TITLE
Immich's 2nd Birthday Q&A - 3rd February @ 16:00UTC

### DISCUSSION THREAD

**Discussion by zackpollard** [immich-app/immich#6867] [Announcements]
Hey everyone! Tomorrow, the 3rd February @ 16:00UTC it is officially Immich's 2nd birthday! To celebrate this, we will be hosting an informal Q&A and chat session with some of the members of the team on our discord (invite link below)! We plan for this to run for around an hour, or until the questions dry up. Please submit any questions you have as well as upvote questions you want answered the most here: http://tinyurl.com/27tzwutt. We look forward to chatting and answering all of your questions tomorrow! Register your interest on the event and discord (should) remind you tomorrow when we go live! 😄 https://discord.gg/CbtCKzYQ?event=1203093679770701834

**Comment by bmachek** [comment]
Absolutely incredible what you guys did in those 2 years!!

**Comment by bo0tzz** [comment]
The recording can now be viewed on youtube: https://www.youtube.com/watch?v=0fE1eJ7fR8o
